Sprint_Tests/vova_timonin.md

```diff
@@ -0,0 +1,23 @@
+1. Раньше он хорошо разбирался в верстке и работе на передовых технологиях на фронте. После небольшого перерыва в несколько лет и глубокой депрессии плюс прослушивания тяжелой музыки, он слабо разбирается в современных тенденциях и много пьет. Если бы он справился с депрессией после смерти жены и ребенка, у него бы сейчас не стоял вопрос поиска работы с релевантной зарплатой.
+
+Previously, he was good in web design and advanced technologies of the front-end. After a short break for several years and deep depression plus hard music listening he is weak in modern trends and drinks a lot. If he had coped with depression after deathes of his wife and child he wouldn't have a question of search a relevant salary job.    
+
+2. Будь у него цель и адекватные зарплатные ожидания он бы уже работал на удаленке где нибудь в штатах или на Бали с перспективой перебраться на Канарские острова и оформить испанское гражданство. Он с детства мечтал жениться на испанке и рано выйти на пенсию и разводить виноград на побережье Балеарского моря.
+
+If he had a target and adequate salary expectations he would already have a remote work anywhere in USA or Bali with perspective of a relocation to Canary and obtaining Spanish citizenship. Since childhood he dreamed of marrying a Spanish girl and early retirenment and growing grapes on the coast of Balearic Sea.
```

Comment:
the USA
with the perspective
the Canary Islands
Since childhood he has dreamt
retirement
the Balearic Sea